qa/cephfs/mount/kclient/overrides/distro/random/flavor/ubuntu_latest.yaml

```diff
@@ -0,0 +1 @@
+.qa/distros/supported/ubuntu_latest.yaml
```

Comment:
Do we need to test two versions of Ubuntu?

Reply:
removed testing ubuntu_16.04